skyportal/facility_apis/interface.py

```diff
@@ -0,0 +1,33 @@
+from ._base import _Base
+
+
+class FollowUpAPI(_Base):
+    """An interface that User-contributed remote facility APIs must provide."""
+
+    # subclasses *must* implement the method below
+    @staticmethod
+    def submit(request):
```

Comment:
This should be documented so that subclasses know what `request` are.

Reply:
fixed

Reply:
fixed